[docs] update EAS Build docs (#13072)
expo/expo: docs/pages/build-reference/build-configuration.md

}
```

> **Note:** If you have a managed workflow project, you'll see `"workflow": "managed"` in the configuration.

This is your EAS Build configuration. It defines a single build profile named `release` (you can have multiple build profiles like `release`, `debug`, `testing`, etc.) for each platform. In the generated configuration, each profile declares that the project is a generic React Native project (unlike a managed Expo project which doesn't contain native code in the project tree). If you want to learn more about `eas.json` see the [Configuration with eas.json](/build/eas-json.md) page.

#### 3. Configure the project

This step varies depending on the project type you have.

#### 3.1. Generic project

> If you choose to configure the Android project, EAS CLI will update your Gradle project so we could build it on our servers.

Reviewer: ```suggestion
If you choose to configure the Android project, EAS CLI will update your Gradle project so we can build it on our servers.
```

> This step patches `build.gradle` by including there our custom signing configuration. The configuration itself is saved to a separate file: `eas-build.gradle`.

Reviewer: ```suggestion
This step patches `build.gradle` and includes our custom signing configuration. The configuration itself is saved to a separate file: `eas-build.gradle`.
```

If you also choose to configure the iOS project, there are no additional steps.

#### 3.2. Managed project

If you haven't configured your `app.json` with `android.package` and/or `ios.bundleIdentifier` yet, EAS CLI will prompt you to specify them.

- `android.package` will be used as the Android application id which is used to identify your app on the Google Play Store
- `ios.bundleIdentifier` will be used to identify you app on the Apple App Store

<ImageSpotlight alt="Application identifier prompts in eas build:configure" src="/static/images/eas-build/configure/02-configure-app-ids.png" containerStyle={{ paddingBottom: 0 }} />

In the example above, we defined exactly the same Android application id and iOS bundle identifier. However, they don't need to match.

#### 5. Next steps

That's all there is to configuring a project to be compatible with EAS Build.

> However, there's one final step. You'll be prompted to commit all the changes we made for you. You can choose to review them before committing. Also, specify the git commit message or use the default one.

Reviewer: ```suggestion
There is one final step — you'll be prompted to commit all the changes we made for you. You can choose to review them before committing, and you can either specify the git commit message or use a default message.
```
